JuanDiegoGuerra/RoR_Capstone-Budget_App
# RoR Capstone Project - Testing & styling

### Acitivity Requirementes - Testing & styling

- Added `rspec` and `capybara` for unit and integration tests.
- Added `tests` for:
  - Group, Purchase and User Models.
  - Groups views (index, show, new).
  - Purchase views (new).
  - Splash screen.
- Add `search` and `left-arrow` buttons for Navbar.
- Added `styles` for:
  - Register and Sign in pages.
  - Notice and alert messages.
- Styling for the pages. ✅
- All Linters are passing! ✅
